src/libs/actions/Report.ts

```diff
@@ -1142,12 +1142,12 @@ function navigateToAndOpenReport(
     const report = isEmptyObject(chat) ? newChat : chat;
 
     // We want to pass newChat here because if anything is passed in that param (even an existing chat), we will try to create a chat on the server
-    openReport(report?.reportID ?? '', '', userLogins, newChat, undefined, undefined, undefined, avatarFile);
+    openReport(report?.reportID, '', userLogins, newChat, undefined, undefined, undefined, avatarFile);
     if (shouldDismissModal) {
         Navigation.dismissModalWithReport(report);
     } else {
         Navigation.navigateWithSwitchPolicyID({route: ROUTES.HOME});
-        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(report?.reportID ?? '-1'), actionType);
+        Navigation.navigate(ROUTES.REPORT_WITH_ID.getRoute(report?.reportID), actionType);
```

Comment:
Make sure it's not called if there is no `report?.reportID`

Reply:
@VickyStash 

But if I prevent navigation from happening it will disrupt normal user flow 🤔 Shouldn't we warn the user that there was some problem, or something?

Reply:
Let's try to test it as I mentioned [here](https://github.com/Expensify/App/pull/54306#discussion_r1893607263). What happens if you navigate user to the screen with `-1` id

Reply:
@VickyStash if the id is invalid, the page will show a Not found page but it seems that all reports in Onyx are deleted